dataland-frontend/tests/e2e/fixtures/p2p/P2pPreparedFixtures.ts

```diff
@@ -0,0 +1,84 @@
+ * @param undef_probability probability for a field to be undefined
+ * @param toggleRandomSectors determines if the sector list should include all possible sectors or a randomized selection
+ * @returns the prepared fixtures
+ */
+export function generateP2pPreparedFixtures(
+  undef_probability = 0.5,
+  toggleRandomSectors = true
+): Array<FixtureData<PathwaysToParisData>> {
+  const preparedFixtures = [];
+  preparedFixtures.push(
+    manipulateFixtureForSixP2pDataSetsInDifferentYears(generateP2pFixture(1, undef_probability, toggleRandomSectors)[0])
+  );
+  preparedFixtures.push(manipulateFixtureForDate(generateP2pFixture(1, undef_probability)[0], "2023-04-18"));
+  preparedFixtures.push(manipulateFixtureForDate(generateP2pFixture(1, undef_probability)[0], "2023-06-22"));
+  preparedFixtures.push(manipulateFixtureForDate(generateP2pFixture(1, undef_probability)[0], "2022-07-30"));
+  preparedFixtures.push(
+    manipulateFixtureForOneP2pDataSetWithTwoSectors(generateP2pFixture(1, undef_probability, toggleRandomSectors)[0])
+  );
+  return preparedFixtures;
+}
+
+/**
+ * Sets the company name and the date in the fixture data to a specific string
+ * @param input Fixture data to be manipulated
+ * @returns the manipulated fixture data
+ */
+function manipulateFixtureForSixP2pDataSetsInDifferentYears(
+  input: FixtureData<PathwaysToParisData>
+): FixtureData<PathwaysToParisData> {
+  input.companyInformation.companyName = "six-p2p-data-sets-in-different-years";
+  if (input.t.general?.general?.dataDate) input.t.general.general.dataDate = "2023-01-01";
+  else console.error("fakeFixture created improperly: dataDate missing");
+  input.reportingPeriod = "2023";
+  return input;
+}
+
+/**
+ * Sets the company name, and, data date and reporting period in the fixture data to
+ * specific values needed for tests.
+ * @param input Fixture data to be manipulated
+ * @param date the date in the format "YYYY-MM-DD"
+ * @returns the manipulated fixture data
+ */
+function manipulateFixtureForDate(
+  input: FixtureData<PathwaysToParisData>,
+  date: string
+): FixtureData<PathwaysToParisData> {
+  input.companyInformation.companyName = "P2p-date-" + date;
+  input.t.general.general.dataDate = date;
+  input.reportingPeriod = date.split("-")[0];
+  return input;
+}
+
+/**
+ * Sets the company name in the fixture data to a specific string, the field "employeeUnder18Apprentices" to "No", and
+ * sets exactly two production sites for the "listOfProductionSites" field.
+ * @param input Fixture data to be manipulated
+ * @returns the manipulated fixture data
+ */
+function manipulateFixtureForOneP2pDataSetWithTwoSectors(
+  input: FixtureData<PathwaysToParisData>
+): FixtureData<PathwaysToParisData> {
+  input.companyInformation.companyName = "one-p2p-data-set-with-two-sectors";
+  input.t.general.general.dataDate = "2022-01-01";
+  input.reportingPeriod = "2022";
+  input.t.general.general.sector = ["Ammonia", "Cement", "LivestockFarming"];
```

Comment:
There might be more sectors even populated with data. I think the `generateP2pFixture` method does not only fill one sector. Should we change that?

Reply:
As far as i have observed there were only ever those 3 sectors set for this specific test